features/testbot/bearing_param.feature

```diff
@@ -75,7 +75,10 @@ Feature: Bearing parameter
             #| 0    | b  | 189 189  | da          | 180                                   |
             | 0    | 1  | 90 270   | ab,bc,cd,cd    | 0->90,90->0,0->270,270->0           |
             | 1    | d  | 10 10    | bc,bc          | 0->0,0->0                           |
-            | 1    | d  | 90 90    | ab,bc,cd,da,da | 0->90,90->0,0->270,270->180,180->0  |
+            # The target point `a` can belong to either `ab` or `ha` edge.
+            # The latter case leads to "Impossible route between points",
+            # because `ab` and `ha` are in different components
+            #| 1    | d  | 90 90    | ab,bc,cd,da,da | 0->90,90->0,0->270,270->180,180->0 |
```

Comment:
Ah now I understand. Yes we should fix this test case by adding a middle node `2` on `da`. I think I misunderstood the original test, it seems like we want to snap to `da` because cd has bearing 270° (since it is going in the different direction).